libraries/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -351,7 +351,8 @@ export class PhasedScriptAction extends BaseScriptAction<IPhasedCommandConfig> {
         customParametersByName.set(configParameter.longName, parserParameter);
       }
 
-      if (buildCacheConfiguration) {
+      if (buildCacheConfiguration?.buildCacheEnabled) {
```

Comment:
We should probably have some test to ensure that this doesn't happen again, given the recent movement in the area

Reply:
I'm okay with doing that in a separate PR, if it enables us to get this fix released more quickly.